after/ftplugin/haskell/intero.vim

```diff
@@ -3,10 +3,6 @@ if exists('b:did_ftplugin_intero') && b:did_ftplugin_intero
 endif
 let b:did_ftplugin_intero = 1
 
-if !has('patch-7.4.1578')
```

Comment:
It's been moved into `plugin/intero.vim`. In hindsight, it makes sense to have it in both, so I'll re-add it.